src/components/ActiveWalletSection/index.tsx

```diff
@@ -121,11 +121,17 @@ export default function ActiveWalletSection() {
     getActiveWalletSignature,
     getGameplayWallets,
     updateActiveWallet,
+    trackEvent,
     isSignedIn
   } = useQuestWrapper()
 
   const connectorName = String(connector?.name)
 
+  const sharedEventProperties = {
+    walletAddress: connectedWallet,
+    walletConnector: connectorName
+  }
```

Comment:
nitpick but could we move this down closer to where it is used? right above `updateActiveWalletMutation`